extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.autoplay-toggle.js

```diff
@@ -0,0 +1,30 @@
+define('wikia.articleVideo.featuredVideo.jwplayer.autoplayToggle', ['wikia.articleVideo.featuredVideo.autoplay'], function (featuredVideoAutoplay) {
+
+	function addAutoplayToggle($settingsTopbar) {
+		$settingsTopbar
+			.find('.jw-settings-close')
+			.before(
+				'<input type="checkbox" id="featuredVideoAutoplayToggle" class="wds-switch__input"' +
+				(featuredVideoAutoplay.isAutoplayEnabled() ? ' checked>' : '>') +
+				'<label for="featuredVideoAutoplayToggle" class="wds-switch__label">Autoplay</label>'
+			);
+
+		$settingsTopbar
+			.find('.wds-switch__label')
+			.on('mouseup pointerup touchstart', function (event) {
+				event.stopPropagation();
+				featuredVideoAutoplay.toggleAutoplay(!event.target.previousSibling.checked);
+			});
+	}
+
+	return function (playerInstance) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "playerInstance". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)